react/week5/github-search-router/github-search-router/src/components/ListItem.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { UserContext } from '../context/Context';
+
+import Users from './User';
+import Loading from '../loading/Loading';
+import { Link } from 'react-router-dom';
+
+class ListItem extends Component {
+   render() {
+      return (
+         <div className='container'>
+            <UserContext.Consumer>
+               {(value) => {
+                  const { usersArray } = value;
+                  const gitSearch = usersArray.map((item) => (
+                     <Link key={item.id} to={`/userprofil/${item.login}`}>
+                        <Users
+                           key={item.id}
+                           items={item.avatar_url}
+                           userName={item.login}
+                        />
+                     </Link>
+                  ));
+
+                  return !usersArray === undefined &&
```

Comment:
```suggestion
return !usersArray &&
```

When you use `!` in front of a variable, you're coercing it to a boolean! Basically just doing `!usersArray` is this equivalent of writing this: 

```js
if(usersArray !== undefined) {
    return true
}